publish/kafka/schemas.go

```diff
@@ -0,0 +1,13 @@
+package kafka
+
+import (
+	"github.com/grafana/metrictank/conf"
+)
+
+func getSchemas(file string) (*conf.Schemas, error) {
```

Comment:
This function and file can probably go away

Reply:
why can this go away? don't we need this when initializing the `kafka.mtPublisher`, to set its `schemas` property (https://github.com/grafana/metrictank/pull/1608/files#diff-c1de7345bb1a47b5638ddad84e8ae8eeR200)?

Reply:
<img width="567" alt="Screen Shot 2020-01-16 at 1 12 21 PM" src="https://user-images.githubusercontent.com/131809/72550962-dd0fba00-3861-11ea-8138-53206b1bf984.png">

Reply:
removed in 34f4d3884938feb1e9b80211561a684afa5f1379